### PR TITLE
update http client to latest version

### PR DESCRIPTION
**Pull request by deepsmokers**

### CODE REVIEW

File: services/adaptive-learning/client/client.py [hunk 1]
```diff
@@ -1,35 +1,62 @@
...
     req.get_model(job_id, token_file, server_address)
 
 
-def upload_file(file_path, token_file, server_address='https://api.neuralet.io'):
+def kill_job(job_id, token_file, server_address="https://api.neuralet.io"):
+    req.kill_job(job_id, token_file, server_address)
+
+
+def kill_job(job_id, token_file, server_address="https://api.neuralet.io"):
+    req.kill_job(job_id, token_file, server_address)
```
Comment (alpha-carinae29) [inline services/adaptive-learning/client/client.py]: This function is duplicated.

File: services/adaptive-learning/client/neuralet_requests.py [hunk 2]
```diff
@@ -69,58 +70,170 @@ def get_model(job_id, token_file, server_address='https://api.neuralet.io'):
...
+        headers = {
+            "Content-Type": "application/json",
+            "Accept": "application/json",
+            "Authorization": f"Bearer {token}",
+        }
+        print(f"Waiting for {url} ...")
+        r = requests.get(url, headers=headers)
+        if r.status_code == 200:
+            pprint(r.json())
+            return r.json()  # TODO: parse and return
```
Comment (alpha-carinae29) [inline services/adaptive-learning/client/neuralet_requests.py]: Does the TODO comment still valid?

File: services/adaptive-learning/client/neuralet_requests.py [hunk 3]
```diff
@@ -69,58 +70,170 @@ def get_model(job_id, token_file, server_address='https://api.neuralet.io'):
...
+            "Content-Type": "application/json",
+            "Accept": "application/json",
+            "Authorization": f"Bearer {token}",
+        }
+        params = {"page": page_number}
+        print(f"Waiting for {url} ...")
+        r = requests.get(url, headers=headers, params=params)
+        if r.status_code == 200:
+            pprint(pop_classess_from_response(r.json()))
+            return r.json()  # TODO: parse and return
```
Comment (alpha-carinae29) [inline services/adaptive-learning/client/neuralet_requests.py]: Does the TODO comment still valid?

File: services/adaptive-learning/client/neuralet_requests.py [hunk 4]
```diff
@@ -69,58 +70,170 @@ def get_model(job_id, token_file, server_address='https://api.neuralet.io'):
...
```
Comment (alpha-carinae29) [inline services/adaptive-learning/client/neuralet_requests.py]: Does the TODO comment still valid?

File: services/adaptive-learning/client/neuralet_requests.py [hunk 5]
```diff
@@ -69,58 +70,170 @@ def get_model(job_id, token_file, server_address='https://api.neuralet.io'):
...
+        headers = {
+            "Content-Type": "application/json",
+            "Authorization": f"Bearer {token}",
+        }
+        data = {"job_id": str(job_id)}
+        data = json.dumps(data)
+        print(f"Waiting for {url} ...")
+        r = requests.post(url, headers=headers, data=data)
+        if r.status_code == 200:
+            return r.content  # TODO: parse and return
```
Comment (alpha-carinae29) [inline services/adaptive-learning/client/neuralet_requests.py]: Does the TODO comment still valid?

**Review by alpha-carinae29 — COMMENTED** [review]
Thank you @deepsmokers for updating the client.
I left some comments :)

**Comment by deepsmokers** [comment]
@alpha-carinae29 
- duplicate code fragment  deleted
- unnecessary comments removed

**Review by alpha-carinae29 — APPROVED** [review]
Thanks @deepsmokers 
@mhejrati LGTM.

**Pull request merged**